make/test.mk

```
@@ -304,3 +307,11 @@ else
 		sed -e 's|REPLACE_IMAGE|${IMAGE_NAME}|g' ${E2E_REPO_PATH}/deploy/operator.yaml | oc apply -f -
     endif
 endif
+
+.PHONY: display-eval
+display-eval:
+	@echo 'export HOST_NS=$(shell oc get projects -l app=host-operator --output=name --no-headers=true -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}' | sort | tail -n 1)'
```

I think that you could make it shorter and drop the jsonpath as well as the --no-headers param:
```suggestion
	@echo 'export HOST_NS=$(shell oc get projects -l app=host-operator --output=name  | cut -f2 -d"/" | sort | tail -n 1)'
```
but it's only my proposal - you don't have to use it as your solution is fine too ;-)

---

well, I kinda like the jsonpath expression, so I'll keep it as-is for now if it's ok for you ;)

---

removed the `--no-headers=true` arg in 689670f